src/django_components/util/tag_parser.py

```diff
@@ -216,7 +222,7 @@ def take_while(tokens: Sequence[Union[str, Tuple[str, ...]]]) -> str:
                 start_index=start_index,
                 quoted=quoted,
                 spread=is_spread,
-                translation=False,
+                translation=is_translation,
```

Comment:
This is the bug. The detected state about whether a variable in a template tag is a translation or not was not being forwarded.